### PR TITLE
[zh-cn] update scroll event of document

### PR DESCRIPTION
**Pull request by jasonren0403**

### CODE REVIEW

File: files/zh-cn/web/api/document/scroll_event/index.md [hunk 1]
```diff
@@ -1,60 +1,51 @@
...
-        {{DOMxRef("GlobalEventHandlers.onscroll", "onscroll")}}
-      </td>
-    </tr>
-  </tbody>
-</table>
-
-> **备注：** 在 iOS UIWebViews 中，滚动进行时不会触发 `scroll` 事件；只有当滚动结束后事件才会被触发。参见 [Bootstrap issue #16202](https://github.com/twbs/bootstrap/issues/16202)。Safari 和 WKWebViews 则没有这个问题。
+{{APIRef}}
+
+当文档视图滚动后，**`scroll`** 事件就会触发。要检测滚动何时结束，请参阅 {{domxref("Document/scrollend_event", "Document：scrollend 事件")}}。关于元素滚动，请参见 {{domxref("Element/scroll_event", "Element：scroll 事件")}}。
```
Comment (jasonren0403) [inline files/zh-cn/web/api/document/scroll_event/index.md:11]:
```suggestion
当文档视图滚动后，**`scroll`** 事件就会触发。要检测滚动何时结束，请参阅 {{domxref("Document/scrollend_event", "Document：scrollend 事件", "", "1")}}。关于元素滚动，请参见 {{domxref("Element/scroll_event", "Element：scroll 事件", "", "1")}}。
```

File: files/zh-cn/web/api/document/scroll_event/index.md [hunk 2]
```diff
@@ -1,60 +1,51 @@
...
+
+## 事件类型
+
+通用的 {{domxref("Event")}}。
 
 ## 示例
 
 ### Scroll 事件节流
 
 由于 `scroll` 事件可被高频触发，事件处理程序不应该执行高性能消耗的操作，如 DOM 操作。而更推荐的做法是使用 {{DOMxRef("Window.requestAnimationFrame()", "requestAnimationFrame()")}}、{{DOMxRef("setTimeout()")}} 或 {{DOMxRef("CustomEvent")}} 给事件节流，如下所述。
```
Comment (jasonren0403) [inline files/zh-cn/web/api/document/scroll_event/index.md:31]:
```suggestion
由于 `scroll` 事件可被高频触发，事件处理器不应该执行高性能消耗的操作，如 DOM 操作。而更推荐的做法是使用 {{DOMxRef("Window.requestAnimationFrame()", "requestAnimationFrame()")}}、{{DOMxRef("setTimeout()")}} 或 {{DOMxRef("CustomEvent")}} 给事件节流，如下所述。
```

File: files/zh-cn/web/api/document/scroll_event/index.md [hunk 3]
```diff
@@ -1,60 +1,51 @@
 ---
-title: "Document: scroll event"
+title: Document：scroll 事件
+short-title: scroll
```
Comment (yin1999) [inline files/zh-cn/web/api/document/scroll_event/index.md:3]: same as another PR: #17160.

Comment (jasonren0403) [reply files/zh-cn/web/api/document/scroll_event/index.md:3]:
```suggestion
```

**Review by yin1999 — APPROVED** [review]
LGTM. Thanks @jasonren0403

**Pull request merged**